ci-operator/step-registry/3scale/apimanager/install/3scale-apimanager-install-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: 3scale-apimanager-install
+  from: 3scale-runner
+  commands: 3scale-apimanager-install-commands.sh
+  resources:
+    requests:
+      cpu: '1'
+      memory: 500Mi
```

Comment:
is this enough resources?

Reply:
the ref is working and creating the apimanager so I would say this is enough resources